apps/mull-ui/src/app/pages/direct-message/direct-message.scss

```
@@ -0,0 +1,6 @@
+@import '../../../variables.scss';
+.direct-messages-container {
+  .page-container {
+    padding: 4rem;
+  }
```

The .header is to get rid of the shadow of the original top nav bar, so we can get what was shown in the picture below

```suggestion
  .page-container {
    padding: 4rem 2rem;
  }  
  .header {
    top: 3.4rem;
    z-index: 4;
  }
  
```

---

agree. I would put 1rem instead of 2 in Cindy suggestion or just have padding-top and no need to add the 2rem.
`  .page-container {
    padding-top: 4rem;
  }`